Vagrantfile

```diff
@@ -0,0 +1,61 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# This script provisions a vagrant development environment with local
+# postgres and redis servers for development and testing.
+PROVISION = <<SCRIPT
+set -e
+
+# cd to /vagrant on login
+cd /vagrant
+grep -q 'cd /vagrant' ~/.bashrc || echo 'cd /vagrant' >> ~/.bashrc
+
+# Install system packages
+sudo apt-get update --quiet
+sudo apt-get install -y $(cat debian_packages.lst) postgresql
```

Comment:
@omarkhan I liked how @brousch put this type of steps in the `Makefile` - it would allow to factorize duplications between here and the `README` file. You would still have to also install `postresql`, which I agree is better left out of the dependencies, but that can be kept separate (or with two targets, like `install_dependencies` and `install_db_dependencies`).

This is a nit though, or it might be something @brousch could add -- I don't feel too strongly about it.